docs/why-netdata.md

```diff
@@ -0,0 +1,201 @@
+-   Capacity planning
+
+However, **slowdowns are 10 times more common** compared to outages (check slide 14 of [Online Performance is Business Performance ](https://www.slideshare.net/KenGodskind/alertsitetrac) reported by Trac Research/AlertSite). Designing a monitoring system targeting only outages and capacity planning solves just a tiny part of the operational problems we face. Check also [Downtime vs. Slowtime: Which Hurts More?](https://dzone.com/articles/downtime-vs-slowtime-which-hurts-more).
+
+All metrics are important, and all metrics should be available when you need them.  For example, to troubleshoot a slowdown, all the metrics are needed, since the real cause of a slowdown is probably quite complex. If we knew the possible reasons, chances are we would have fixed them before they become a problem.
+
+#### What do others do?
+
+Most monitoring solutions, when they can detect something, provide just a hint (e.g., _"Hey, there is a 20% drop in requests per second over the last minute"_), and they expect us to use the console to determine the root cause.  But this introduces a problem with the console because how do you troubleshoot a slowdown if the slowdown lifetime is just a few seconds, randomly spread throughout the day?
+
+You can't! You end up spending your entire day on the console, waiting for the problem to happen again while logged in. A blame war starts: developers blame the systems, sysadmins blame the hosting provider, someone says it is a DNS problem, another one believes it is network related, and so on. We have all experienced this multiple times!
+So, why do monitoring solutions and SaaS providers filter out metrics?
+
+**They can't do otherwise!**
+
+1.  Centralization of metrics depends on metrics filtering, to control monitoring costs. Time-series databases limit the number of metrics collected because the number of metrics influences their performance significantly. They get congested at scale.
+2.  It is a lot easier to provide an illusion of monitoring by using a few basic metrics.
+3.  Troubleshooting slowdowns is the hardest IT problem to solve, so most solutions avoid it.
+
+#### What does Netdata do?
+
+Netdata collects, stores, and visualizes everything, every single metric exposed by systems and applications. The console should not provide more metrics.  Because of Netdata's distributed nature, the number of metrics collected does not have any noticeable effect on the performance or the cost of the monitoring infrastructure.
+
+Of course, since Netdata is also about [meaningful presentation](meaningful-presentation.md), the number of metrics makes Netdata development slower.  So, we need to have a good understanding of the metrics before adding them to Netdata. We organize the metrics, add information related to them, configure alarms for them, so that you, the Netdata users, have the best out-of-the-box experience and all the information required to kill the console for troubleshooting slowdowns.
```

Comment:
```suggestion
Of course, since Netdata is also about [meaningful presentation](#meaningful-presentation), the number of metrics makes Netdata development slower.  So, we need to have a good understanding of the metrics before adding them to Netdata. We organize the metrics, add information related to them, configure alarms for them, so that you, the Netdata users, have the best out-of-the-box experience and all the information required to kill the console for troubleshooting slowdowns.
```